meinberlin/templates/account/password_reset_done.html

```diff
@@ -0,0 +1,19 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+{% load allauth %}
+{% load account %}
+
+{% block head_title %} 
+    {% translate "Password Reset" %}
+{% endblock head_title %}
+
+{% block content %}
+    <h1>{% translate "Password Reset" %}</h1>
+    {% if user.is_authenticated %}
+        {% include "account/snippets/already_logged_in.html" %}
+    {% endif %}
+    <p>
+        {% blocktrans %}We have sent you an email. If you have not received it please check your spam folder. Otherwise contact us if you do not receive it in a few minutes.{% endblocktrans %}
```

Comment:
we have a custom german translation for this but it isn't picked up anymore because there was a wording change I think. So either we change `email` to `e-mail` here or change it the other way round in`meinberlin/apps/contrib/django_standard_messages.py`.

Reply:
I do see the German translation though locally
![image](https://github.com/liqd/a4-meinberlin/assets/5695572/021d5040-ee3f-49b1-9a00-39d0dbeea885)

Reply:
yes, that's the one provided by allauth / django, if you check on stage: https://meinberlin-stage.liqd.net/accounts/password/reset/done/ the wording is different (more formal german)

Reply:
oh ok, I got what you mean now.
I will change the english

Reply:
If I add what we have in django messages in the template doesn't work, then it doesn't render into German at all. I saw that the django translation in locales, it is using the original msg of the template.

Reply:
@goapunk any idea about that above? Can't make it to work.